src/main/java/org/takes/rq/RqHeaders.java

```diff
@@ -89,7 +90,9 @@ public Base(final Request req) {
         public List<String> header(final CharSequence key)
             throws IOException {
             final List<String> values = this.map().getOrDefault(
-                new EnglishLowerCase(key).string(),
+                new UncheckedText(
+                    new Lowered(key.toString())
```

Comment:
@fanifieiev Isn't `key` already `String`? Why use `toString()` here?

Reply:
@fabriciofx Because key is CharSequence